config/vufind/config.ini

```diff
@@ -165,6 +165,17 @@ allowSavedSearches = true
 nonJavascriptSupportEnabled = false
 ; Generator value to display in an HTML header <meta> tag:
 generator = "VuFind 8.0.1"
+; ExpandedHeadTitle style version
+; pre = set the information as prefix before header
+; post = set the information as postfix after header
+; both = set both (prefix and postfix)
+expandedHeadTitle_style = false
+; set separator for title in header
+;expandedHeadTitle_sep = " | "
```

Comment:
I'd add a note that the prefix gets prepended or appended directly to the postfix or prefix and should include any spaces needed around it.